### ISSUE TITLE
Topology Pipelines - Focussed element should contain the border

### ISSUE THREAD

**Issue by karthikjeeyar** [patternfly/patternfly-react#8148]
**Describe the problem**

This is a A11y Issue, focussing on a task via keyboard doesn't have any border to highlight the node.

**How do you reproduce the problem?**

https://topology-pipeline-zoom2.surge.sh/

**Expected behavior**
Focussed element should have a border to highligh

**Screenshots**
Previous version:
![image](https://user-images.githubusercontent.com/9964343/193829278-c155d2ea-a17a-449c-bdbf-9cab16156889.png)

PF version:

![image](https://user-images.githubusercontent.com/9964343/193829339-5d22fab9-5b4f-49a5-bdda-1dbf727fb4a9.png)

**What is your product and what release date are you targeting?**
Openshift Devconsole, HACBS

cc: @serenamarie125 @jeff-phillips-18

**Comment by jeff-phillips-18** [comment]
Nodes that need a focus border can be created by the application. Add an anchor around the node and update the styling to show the focus border.  We _could_ add a `focusable` option to the node if desired but it could just be handled by the app.
